command/exclude.go

```diff
@@ -0,0 +1,44 @@
+package command
+
+import (
+	"regexp"
+	"strings"
+)
+
+func wildCardToRegexp(pattern string) string {
+	patternRegex := regexp.QuoteMeta(pattern)
+	patternRegex = strings.Replace(patternRegex, "\\?", ".", -1)
+	patternRegex = strings.Replace(patternRegex, "\\*", ".*", -1)
+	patternRegex = patternRegex + "$"
+	return patternRegex
+}
+
+// createExcludesFromWildcard creates regex strings from wildcard.
+func createExcludesFromWildcard(inputExcludes []string) []*regexp.Regexp {
+	result := make([]*regexp.Regexp, 0)
+	for _, input := range inputExcludes {
+		if input != "" {
+			regexVersion := wildCardToRegexp(input)
+			regexpCompiled, err := regexp.Compile(regexVersion)
+			if err != nil {
+				continue
+			}
```

Comment:
I think the error should be handled.

Reply:
I think If the regex is not compiled, then we can ignore it. How should i handle this error?

Reply:
It would not nice to users to ignore it silently IMO. How returning it and aborting command execution sounds like to you?

Reply:
now errors are returned and aborting is completed.